python/build_wheel.sh.in

```diff
@@ -36,8 +36,8 @@ if [ "${PYTHON2_DETECTED}" == 1 ]; then
     cd ${BUILD_DIR}
     virtualenv -p "$(command -v python)" build/venv2
     source build/venv2/bin/activate
-    pip install --upgrade setuptools pip wheel
-    python ${SOURCE_DIR}/setup.py bdist_wheel
+    build/venv2/bin/python build/venv2/bin/pip install --upgrade setuptools pip wheel
```

Comment:
Better define the path as a bash variable ?

Reply:
Since we're calling `build/venv2/bin/activate` above, the `pip` and `python` executables should refer to the ones from `venv2`. What system isn't this working on for you? Is your `venv2` directory so deeply nested?

Perhaps we could use this instead, to avoid hard-coding `build/venv2/bin` multiple times?
```python -m pip```
at worse:
`build/venv2/bin/python -m pip`